letsencrypt/client/crypto_util.py

```diff
@@ -189,3 +190,43 @@ def get_cert_info(filename):
         "serial": cert.get_serial_number(),
         "pub_key": "RSA " + str(cert.get_pubkey().size() * 8),
     }
+
+
+def get_sans_from_csr(csr):
+    """Get list of Subject Alternative Names from signing request.
+
+    :param str csr: Certificate Signing Request in PEM format
+
+    :returns: List of referenced subject alternative names
+    :rtype: list
+    """
+    # TODO: This is a temporary solution involving parsing the .as_text()
+    #       output because there doesn't seem to be a built-in feature in
+    #       any Python cryptography module that performs this function.
+    #       In the future we should try to replace this with a more direct
+    #       use of relevant OpenSSL or other X509-parsing APIs.
+    req = M2Crypto.X509.load_request_string(csr)
+    text = req.as_text().split("\n")
+    if len(text) < 2 or text[0] != "Certificate Request:" or \
```

Comment:
please don't use `\` at the end of the line, but use implicit line continuation inside braces